federator/src/contracts/IBridgeV3.ts

```diff
@@ -21,6 +24,10 @@ export class IBridgeV3 implements IBridge {
   }
 
   async getPastEvents(eventName: string, destinationChainId: number, options: any): Promise<EventData[]> {
+    if ([4, 56, 97].includes(this.chainId)) {
```

Comment:
Perhaps is a good idea to get the array of chain ids from a constant/s?

Reply:
Will do that 👍